### DISCUSSION TITLE
Color of color picker window not the same as picked color

### DISCUSSION THREAD

**Discussion by ghost** [Toinane/colorpicker#96] [Q&A] [answered]
When I pick a color, the hex value is correct. I can check this in other image editors or websites. However, the color shown in the picker itself is not matching the picked color aka the hex value. It's very noticeable with red.
![red](https://user-images.githubusercontent.com/3965318/106746962-8e2af300-6623-11eb-9a32-2ca7f5428f5e.png)

Has this to do with color space? Can I change this in the app?

**Comment by Toinane** [comment, marked as answer]
Hey, sorry I didn't see your message before today! Yes it's an issue with colorspace. The module I use for eyedropper didn't work with others color spaces. I need to create my own module for that, its in work
